core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -58,7 +58,8 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
     override val workbenchMetricBaseName: String = "test",
     val requesterPaysRole: String = requesterPaysRole,
     val useWorkflowCollectionField: Boolean = false,
-    val useWorkflowCollectionLabel: Boolean = false) extends WorkflowSubmission {
+    val useWorkflowCollectionLabel: Boolean = false,
+    val defaultBackend: String = "PAPIv2") extends WorkflowSubmission {
```

Comment:
`String` is very easy to have typo and compiler won't help much to ensure it's correct..Would be nice if you can make defaultBackend an ADT, for instance
```
sealed abstract class CromwellBackend{
    def asString: String
}
object CromwellBackend {
   final case class JES extends CromwellBackend {def asString: String = "JES"}
   final case class PapiV2  extends CromwellBackend {def asString: String = "PAPIv2"}
}
```

This way, you won't need to write `PAPIv2` everywhere and worrying about at some place you might have `Papiv2`.

What's even better is moving this to workbench-libs model module, cuz I'm sure other services can benefit from it.